pages/token-unit-conversion.tsx

```diff
@@ -69,14 +85,16 @@ function UnitElements({ units, onChange, setDecimal }: UnitElementsProps): JSX.E
             <tr>
               <th
                 scope="col"
-                className="py-1 px-6 text-xs font-medium tracking-wider text-left text-gray-700 uppercase dark:text-gray-400"
```

Comment:
is this a pure formatting change? these files should be formatted with prettier. Can you double check configuration?